From 7a6d75ba3756fcd60a78d43dc314f9f455ff2f62 Mon Sep 17 00:00:00 2001
From: Sai Surya Teja <sssteja@habana.ai>
Date: Tue, 02 Nov 2021 17:19:41 +0200
Subject: [PATCH] [SW-59942] fit_load_type: edit misleading log

Modifying log "load type does not exist!" to be more informative than 
error.

Change-Id: I6b6fdf6453e7d0e764041d6bf35e2c71902e3fae
---

diff --git a/lib/preboot/fit.c b/lib/preboot/fit.c
index 0724a14..56d5d57 100644
--- a/lib/preboot/fit.c
+++ b/lib/preboot/fit.c
@@ -1052,7 +1052,7 @@
 	if (conf >= 0) {
 		cell = fdt_getprop(fit, conf, FIT_LOAD_TYPE_PROP, &len);
 		if (!cell) {
-			preboot_puts("load type does not exist!\n");
+			preboot_puts("No load type. Assuming old FIT version\n");
 			*load_type = type;
 			return 0;
 		}
